app/views/questions/edit.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Edit a question</h1>
```

Comment:
Will we ever need this file?

Reply:
No, I actually deleted it in a later commit